mp/src/game/shared/momentum/util/mom_util.cpp

```diff
@@ -68,34 +68,102 @@ void MomUtil::DispatchConCommand(const char *pszCommand)
 }
 #endif
 
+static void MountGames(bool &gotCS, bool &gotTF)
+{
+    KeyValuesAD gameinfo("GameInfo");
+    gameinfo->LoadFromFile(filesystem, "gameinfo.txt");
+
+    char path[MAX_PATH];
+    if (KeyValues *pMounts = gameinfo->FindKey("mount"))
```

Comment:
This should instead be an early return.

```suggestion
    const auto pMounts = gameinfo->FindKey("mount");
    if (!pMounts)
        return;
```